src/internal/effectable.ts

```diff
@@ -69,7 +69,19 @@ export const EffectPrototype: Effect.Effect<never, never, never> = {
 
 /** @internal */
 export const CommitPrototype = {
```

Comment:
```ts
/** @internal */
export const CommitPrototype: Effect.Effect<never, never, never> = {
  ...EffectPrototype,
  _op: OpCodes.OP_COMMIT
} as any
```

Maybe something like this.

Reply:
Yea... That works. No idea why I didn't think of that, lol. Thanks @tim-smart!